src/SonarQube.Client.Tests/Api/SonarQubeService_GetSuppressedIssuesAsync.cs

```diff
@@ -73,176 +73,229 @@ public async Task GetSuppressedIssuesAsync_From_7_20()
         {
             await ConnectToSonarQube("7.2.0.0");
 
-            SetupRequest("api/issues/search?projects=simplcom&statuses=RESOLVED&p=1&ps=500", @"
+            SetupRequest("api/issues/search?projects=com.github.kevinsawicki:http-request&statuses=RESOLVED&p=1&ps=500", @"
```

Comment:
Created a new version of the response with more issues and containing all fields in the JSON object (e.g. `components`).